shared/css/hamburger.scss

```diff
@@ -256,3 +255,154 @@
     }
   }
 }
+
+#help-button {
```

Comment:
can this be merged with `#hambergur`?

Reply:
I merged some of this. There are some differences, though I was able to merge help-contents with hamburger-contents completely.

I'm not sure about the styling of this now. @Hamms or @breville any tips/opinions on how to best structure this file?

Reply:
I'd do something like this:

```diff
diff --git a/shared/css/hamburger.scss b/shared/css/hamburger.scss
index be375af6898..a1af4aeac09 100644
--- a/shared/css/hamburger.scss
+++ b/shared/css/hamburger.scss
@@ -38,21 +38,14 @@
   position: relative;
   font-size: 14px;
 
-  #hamburger-icon {
+  #hamburger-icon, #help-icon {
     cursor: pointer;
-    padding: 10px 35px 16px 0;
     height: 20px;
     vertical-align: middle;
     text-decoration: none;
-    float: right;
-    margin-top: 8px;
-    display: inline-block;
 
-    %extend_burger {
+    %extend_icon {
       cursor: pointer;
-      height: 3px;
-      width: 25px;
-      border-radius: 2px;
       background: $white;
       position: absolute;
       display: block;
@@ -61,7 +54,7 @@
       margin-bottom: 2px;
     }
     span {
-      @extend %extend_burger;
+      @extend %extend_icon;
       transition: all 500ms ease-in-out;
       html[dir=rtl] & {
         margin-right: -25px;
@@ -73,12 +66,12 @@
         }
       }
       &::before {
-        @extend %extend_burger;
+        @extend %extend_icon;
         transition: all 500ms ease-in-out;
         top: -10px;
       }
       &::after {
-        @extend %extend_burger;
+        @extend %extend_icon;
         transition: all 500ms ease-in-out;
         bottom: -10px;
       }
@@ -256,62 +249,28 @@
   }
 }
 
+#hamburger {
+  #hamburger-icon {
+    padding: 10px 35px 16px 0;
+    float: right;
+    margin-top: 8px;
+    display: inline-block;
+
+    span {
+      height: 3px;
+      width: 25px;
+      border-radius: 2px;
+    }
+  }
+}
+
 #help-button {
   padding-left: 20px;
   padding-top: 4px;
 
   #help-icon {
-    cursor: pointer;
     padding: 9px 0 16px 0;
-    height: 20px;
     font-size: 26px;
-    vertical-align: middle;
-    text-decoration: none;
     color: $white;
-
-    %extend_help {
-      cursor: pointer;
-      background: $white;
-      position: absolute;
-      display: block;
-      content: '';
-      margin-top: 2px;
-      margin-bottom: 2px;
-    }
-    span {
-      @extend %extend_help;
-      transition: all 500ms ease-in-out;
-      html[dir=rtl] & {
-        margin-right: -25px;
-        &::before {
-          margin-right: 0;
-        }
-        &::after {
-          margin-right: 0;
-        }
-      }
-      &::before {
-        @extend %extend_help;
-        transition: all 500ms ease-in-out;
-        top: -10px;
-      }
-      &::after {
-        @extend %extend_help;
-        transition: all 500ms ease-in-out;
-        bottom: -10px;
-      }
-    }
-    &.active span {
-      background-color: transparent;
-      &::before {
-        top: 0;
-        transform: rotate(45deg);
-      }
-      &::after {
-        top: 0;
-        transform: rotate(-45deg);
-      }
-    }
   }
-
 }
```

Basically, have three sections: one for all the styles shared between the two, one for the hamburger-specific styles, one for the help-specific styles

Reply:
Github e-mails don't show inserted code apparently so I didn't see this until now. :( I'll play around with combining the icons a bit more, though there's a part of me that thinks they're different enough that they are okay being separate. I did move #hamburger-icon under a separate #hamburger section.

Reply:
I further combined these. Looks like the help-button doesn't need the extend_help bit (at least not yet) so I've removed that.